### PR TITLE
Add sleepy_craft_studios sleepy_keeb

### PR DESCRIPTION
**Pull request by CobyPear**
## Description
Adds `sleepy_craft_studios` under `keyboards`, and the `sleepy_keeb` with VIA support

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by dunk2k — CHANGES_REQUESTED** [review]
Add Community Layout support

File: keyboards/sleepy_craft_studios/sleepy_planck/config.h
Comment (dunk2k) [inline keyboards/sleepy_craft_studios/sleepy_planck/config.h:1]: please can this file be deleted? as it provides no functional purpose

File: keyboards/sleepy_craft_studios/sleepy_planck/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,41 @@
+// Copyright 2023 QMK
```
Comment (dunk2k) [inline keyboards/sleepy_craft_studios/sleepy_planck/keymaps/default/keymap.c:1]: could year and name be changed

File: keyboards/sleepy_craft_studios/sleepy_planck/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+     * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐
+     * │esc│ q │ w │ e │ r │ t │ y │ u │ i │ o │ p │ <-│
+     * ├───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┤
+     * │tab│ a │ s │ d │ f │ g │ h │ j │ k │ l │ ; │ ' │
+     * ├───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┤
+     * │lsh│ z │ x │ c │ v │ b │ n │ m │ , │ . │ / │ent│
+     * ├───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┤
+     * │ ` │ctl│alt│gui│mo1│ space │mo2│up │dwn│lft│rgt│
+     * └───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┘
+     */
```
Comment (dunk2k) [inline keyboards/sleepy_craft_studios/sleepy_planck/keymaps/default/keymap.c:23]:
```suggestion
```

File: keyboards/sleepy_craft_studios/sleepy_planck/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+     * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐
+     * │esc│ q │ w │ e │ r │ t │ y │ u │ i │ o │ p │ <-│
+     * ├───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┤
+     * │tab│ a │ s │ d │ f │ g │ h │ j │ k │ l │ ; │ ' │
+     * ├───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┤
+     * │lsh│ z │ x │ c │ v │ b │ n │ m │ , │ . │ / │ent│
+     * ├───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┼───┤
+     * │ ` │ctl│alt│gui│mo1│ space │mo2│up │dwn│lft│rgt│
+     * └───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┘
+     */
```
Comment (dunk2k) [inline keyboards/sleepy_craft_studios/sleepy_planck/keymaps/via/keymap.c:23]:
```suggestion
```

File: keyboards/sleepy_craft_studios/sleepy_planck/readme.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# sleepy_planck
+
+![sleepy_planck](https://bucket.sleepycraftstudios.com/images/sleepy_planck_complete_blue_silver_silk-rotated-270.webp)
```
Comment (dunk2k) [inline keyboards/sleepy_craft_studios/sleepy_planck/readme.md:3]: Please can this image be (re-)hosted on Imgur?

File: keyboards/sleepy_craft_studios/sleepy_planck/info.json
Comment (dunk2k) [inline keyboards/sleepy_craft_studios/sleepy_planck/info.json:1]: Does this keyboard actually use a pro micro or elite-c development board?

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/sleepy_craft_studios/sleepy_planck/readme.md (hunk 4 above)
Comment (CobyPear) [reply keyboards/sleepy_craft_studios/sleepy_planck/readme.md:3]: Sure thing

File: keyboards/sleepy_craft_studios/sleepy_planck/info.json
Comment (CobyPear) [reply keyboards/sleepy_craft_studios/sleepy_planck/info.json:1]: Yes, those are the two boards I have personally tested.

File: keyboards/sleepy_craft_studios/sleepy_planck/info.json [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+        "command": false,
+        "extrakey": false,
+        "mousekey": false,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["D7", "E6", "B4", "B5", "F4", "F5", "F6", "F7", "B1", "B3", "B2", "B6"],
+        "rows": ["D1", "D0", "D4", "C6"]
+    },
+    "processor": "atmega32u4",
```
Comment (dunk2k) [inline keyboards/sleepy_craft_studios/sleepy_planck/info.json:18]:
```suggestion
    "diode_direction": "COL2ROW",
    "features": {
        "bootmagic": true,
        "command": false,
        "extrakey": false,
        "mousekey": false,
        "nkro": true
    },
    "development_board": "promicro",
    "matrix_pins": {
        "cols": ["D7", "E6", "B4", "B5", "F4", "F5", "F6", "F7", "B1", "B3", "B2", "B6"],
        "rows": ["D1", "D0", "D4", "C6"]
    },
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/sleepy_craft_studios/sleepy_planck/readme.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+# sleepy_planck
+
+![sleepy_planck](https://i.imgur.com/y3cessF.jpeg)
+
+The Sleepy Planck is a hand-wired keyboard based on the 4x12 Planck layout. The Sleepy Planck integrates hot-swap socket and diode holders into the plate for easier building.
+
+- Keyboard Maintainer: [Sleepy Craft Studios](https://github.com/sleepy-craft-studios)
+- Hardware Supported: Pro-Micro and clones, Elite-C
```
Comment (dunk2k) [inline keyboards/sleepy_craft_studios/sleepy_planck/readme.md:8]:
```suggestion
- Hardware Supported: Pro-Micro (and clones/adapations of)
```

File: keyboards/sleepy_craft_studios/sleepy_planck/readme.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+# sleepy_planck
```
Comment (dunk2k) [inline keyboards/sleepy_craft_studios/sleepy_planck/readme.md:1]:
```suggestion
# Sleepy Planck
```

File: keyboards/sleepy_craft_studios/sleepy_planck/keymaps/via/keymap.c [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+        KC_TILD, KC_EXLM, KC_AT, KC_HASH, KC_DLR, KC_PERC, KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, KC_TRNS,
+        KC_DEL, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_UNDS, KC_PLUS, KC_LCBR, KC_RCBR, KC_PIPE,
+        KC_TRNS, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, LSFT(KC_NUHS), LSFT(KC_NUBS), KC_HOME, KC_END, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_MNXT, KC_VOLD, KC_VOLU, KC_MPLY),
+
+    [_FN] = LAYOUT_planck_mit(
+        KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, RGB_TOG,
+        KC_DEL, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_MINS, KC_EQL, KC_LBRC, KC_RBRC, KC_BSLS,
+        KC_TRNS, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, KC_NUHS, KC_NUBS, KC_PGUP, KC_PGDN, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, RGB_MODE_FORWARD, KC_MNXT, KC_VOLD, KC_VOLU, KC_MPLY),
```
Comment (dunk2k) [inline keyboards/sleepy_craft_studios/sleepy_planck/keymaps/via/keymap.c:29]:
```suggestion
        KC_ESC,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_BSPC,
        KC_TAB,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,
        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_ENT,
        KC_GRV,  KC_LCTL, KC_LALT, KC_LGUI, MO(1),        KC_SPC,      MO(2),   KC_UP,   KC_DOWN, KC_LEFT, KC_RGHT
    ),
    [_RAISE] = LAYOUT_planck_mit(
        KC_TILD, KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC, KC_CIRC, KC_AMPR,    KC_ASTR,    KC_LPRN, KC_RPRN, _______,
        KC_DEL,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_UNDS,    KC_PLUS,    KC_LCBR, KC_RCBR, KC_PIPE,
        _______, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  S(KC_NUHS), S(KC_NUBS), KC_HOME, KC_END,  _______,
        _______, _______, _______, _______, _______,     _______,      _______,    KC_MNXT,    KC_VOLD, KC_VOLU, KC_MPLY
    ),
    [_FN] = LAYOUT_planck_mit(
        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    RGB_TOG,
        KC_DEL,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_MINS, KC_EQL,  KC_LBRC, KC_RBRC, KC_BSLS,
        _______, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_NUHS, KC_NUBS, KC_PGUP, KC_PGDN, _______,
        _______, _______, _______, _______, _______,     _______,      RGB_MOD, KC_MNXT, KC_VOLD, KC_VOLU, KC_MPLY
    ),
```

File: keyboards/sleepy_craft_studios/sleepy_planck/keymaps/default/keymap.c [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+        KC_TILD, KC_EXLM, KC_AT, KC_HASH, KC_DLR, KC_PERC, KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, KC_TRNS,
+        KC_DEL, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_UNDS, KC_PLUS, KC_LCBR, KC_RCBR, KC_PIPE,
+        KC_TRNS, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, LSFT(KC_NUHS), LSFT(KC_NUBS), KC_HOME, KC_END, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_MNXT, KC_VOLD, KC_VOLU, KC_MPLY),
+
+    [_FN] = LAYOUT_planck_mit(
+        KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_TRNS,
+        KC_DEL, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_MINS, KC_EQL, KC_LBRC, KC_RBRC, KC_BSLS,
+        KC_TRNS, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, KC_NUHS, KC_NUBS, KC_PGUP, KC_PGDN, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_MNXT, KC_VOLD, KC_VOLU, KC_MPLY),
```
Comment (dunk2k) [inline keyboards/sleepy_craft_studios/sleepy_planck/keymaps/default/keymap.c:29]:
```suggestion
        KC_ESC,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_BSPC,
        KC_TAB,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,
        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_ENT,
        KC_GRV,  KC_LCTL, KC_LALT, KC_LGUI, MO(1),        KC_SPC,      MO(2),   KC_UP,   KC_DOWN, KC_LEFT, KC_RGHT
    ),
    [_RAISE] = LAYOUT_planck_mit(
        KC_TILD, KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC, KC_CIRC, KC_AMPR,    KC_ASTR,    KC_LPRN, KC_RPRN, _______,
        KC_DEL,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_UNDS,    KC_PLUS,    KC_LCBR, KC_RCBR, KC_PIPE,
        _______, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  S(KC_NUHS), S(KC_NUBS), KC_HOME, KC_END,  _______,
        _______, _______, _______, _______, _______,     _______,      _______,    KC_MNXT,    KC_VOLD, KC_VOLU, KC_MPLY
    ),
    [_FN] = LAYOUT_planck_mit(
        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    RGB_TOG,
        KC_DEL,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_MINS, KC_EQL,  KC_LBRC, KC_RBRC, KC_BSLS,
        _______, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_NUHS, KC_NUBS, KC_PGUP, KC_PGDN, _______,
        _______, _______, _______, _______, _______,     _______,      RGB_MOD, KC_MNXT, KC_VOLD, KC_VOLU, KC_MPLY
    ),
```

File: keyboards/sleepy_craft_studios/sleepy_planck/readme.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
+# Sleepy Planck
+
+![sleepy_planck](https://i.imgur.com/y3cessF.jpeg)
+
+The Sleepy Planck is a hand-wired keyboard based on the 4x12 Planck layout. The Sleepy Planck integrates hot-swap socket and diode holders into the plate for easier building.
+
+- Keyboard Maintainer: [Sleepy Craft Studios](https://github.com/sleepy-craft-studios)
+- Pro Micro development board (and clones/adapations of)
```
Comment (dunk2k) [inline keyboards/sleepy_craft_studios/sleepy_planck/readme.md:8]:
```suggestion
- Hardware Supported: Pro Micro development board (and clones/adapations of)
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/sleepy_craft_studios/sleepy_planck/readme.md (hunk 10 above)
Comment (CobyPear) [reply keyboards/sleepy_craft_studios/sleepy_planck/readme.md]: oops, fixed.

**Review by dunk2k — APPROVED** [review]

File: keyboards/sleepy_craft_studios/sleepy_keeb/info.json [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+    "manufacturer": "Sleepy Craft Studios",
+    "keyboard_name": "sleepy_keeb",
+    "maintainer": "Sleepy Craft Studios",
+    "development_board": "promicro",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "extrakey": false,
+        "mousekey": false,
```
Comment (waffle87) [inline keyboards/sleepy_craft_studios/sleepy_keeb/info.json:11]:
```suggestion
        "extrakey": true,
        "mousekey": true,
```

File: keyboards/sleepy_craft_studios/sleepy_keeb/keymaps/default/keymap.c [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+        KC_TILD, KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC, KC_CIRC, KC_AMPR,    KC_ASTR,    KC_LPRN, KC_RPRN, _______,
+        KC_DEL,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_UNDS,    KC_PLUS,    KC_LCBR, KC_RCBR, KC_PIPE,
+        _______, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  S(KC_NUHS), S(KC_NUBS), KC_HOME, KC_END,  _______,
+        _______, _______, _______, _______, _______,     _______,      _______,    KC_MNXT,    KC_VOLD, KC_VOLU, KC_MPLY
+    ),
+    [_FN] = LAYOUT_planck_mit(
+        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    RGB_TOG,
+        KC_DEL,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_MINS, KC_EQL,  KC_LBRC, KC_RBRC, KC_BSLS,
+        _______, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_NUHS, KC_NUBS, KC_PGUP, KC_PGDN, _______,
+        _______, _______, _______, _______, _______,     _______,      RGB_MOD, KC_MNXT, KC_VOLD, KC_VOLU, KC_MPLY
```
Comment (waffle87) [inline keyboards/sleepy_craft_studios/sleepy_keeb/keymaps/default/keymap.c:29]:
```suggestion
        _______, _______, _______, _______, _______,     _______,    _______, KC_MNXT, KC_VOLD, KC_VOLU, KC_MPLY
```

File: keyboards/sleepy_craft_studios/sleepy_keeb/keymaps/via/keymap.c [hunk 13]
```diff
@@ -0,0 +1,31 @@
...
+        KC_TILD, KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC, KC_CIRC, KC_AMPR,    KC_ASTR,    KC_LPRN, KC_RPRN, _______,
+        KC_DEL,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_UNDS,    KC_PLUS,    KC_LCBR, KC_RCBR, KC_PIPE,
+        _______, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  S(KC_NUHS), S(KC_NUBS), KC_HOME, KC_END,  _______,
+        _______, _______, _______, _______, _______,     _______,      _______,    KC_MNXT,    KC_VOLD, KC_VOLU, KC_MPLY
+    ),
+    [_FN] = LAYOUT_planck_mit(
+        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    RGB_TOG,
+        KC_DEL,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_MINS, KC_EQL,  KC_LBRC, KC_RBRC, KC_BSLS,
+        _______, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_NUHS, KC_NUBS, KC_PGUP, KC_PGDN, _______,
+        _______, _______, _______, _______, _______,     _______,      RGB_MOD, KC_MNXT, KC_VOLD, KC_VOLU, KC_MPLY
```
Comment (waffle87) [inline keyboards/sleepy_craft_studios/sleepy_keeb/keymaps/via/keymap.c:29]:
```suggestion
        _______, _______, _______, _______, _______,     _______,   _______, KC_MNXT, KC_VOLD, KC_VOLU, KC_MPLY
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

**Review by waffle87 — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**